### PR TITLE
React Autosuggest Search Bar

### PR DESCRIPTION
**Pull request by AFogarty72**
Add react-autosuggest into search bar (:

**Cannot be merged until new version of react-blur-admin is published.
Has styles that need to be imported.**

### CODE REVIEW

File: src/layout/components/page-top.js [hunk 1]
```diff
@@ -49,8 +50,10 @@ export class PageTop extends React.Component {
   renderSearch() {
     return (
       <div className="search">
-        <i className="ion-ios-search-strong"></i>
-        <input id="searchInput" type="text" placeholder="Search for..." />
+        <SearchBar>
```
Comment (jwdotjs) [inline src/layout/components/page-top.js]: It looks like SearchBar doesn't do anything with `this.props.children` in that component, so there shouldn't be anything inside the SearchBar in this component

File: src/layout/components/search-bar.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      suggestions: this.getSuggestions(value),
+    });
+  }
+
+  getSuggestions(value) {
+    const inputValue = value.trim().toLowerCase();
+    const inputLength = inputValue.length;
+
+    return inputLength === 0 ? [] : suggestionExamples.filter(lang =>
+    lang.text.toLowerCase().slice(0, inputLength) === inputValue
```
Comment (jwdotjs) [inline src/layout/components/search-bar.js]: Spacing on this and next line. And on 76

**Comment by jwdotjs** [comment]
Looks good but we will have loading and no results indicators shortly that we should also incorporate here

**Comment by jwdotjs** [comment]
This story should be unblocked now! Apologies for the wait

**Comment by AFogarty72** [comment]
references #39

**Comment by AFogarty72** [comment]
HUGE credit to @mattlorey for helping me! 💃

**Pull request merged**